lib/livebook/session.ex

```diff
@@ -573,6 +585,23 @@ defmodule Livebook.Session do
     {:reply, :ok, maybe_save_notebook_sync(state)}
   end
 
+  def handle_call(:close, _from, state) do
+    maybe_save_notebook_sync(state)
+    broadcast_message(state.session_id, :session_closed)
+
+    {:stop, :shutdown, :ok, state}
```

Comment:
Why aren't we returning `:normal` given that's an expected termination?

Reply:
You almost never want to return `:normal` and almost ways `:shutdown` as `:shutdown` breaks links and `:normal` does not. None of them are logged though. The docs here explain the differences: https://hexdocs.pm/elixir/Supervisor.html#module-exit-reasons-and-restarts

Reply:
I see, thanks!